app/controllers/protocols_controller.rb

```diff
@@ -21,9 +21,9 @@
 class ProtocolsController < ApplicationController
   respond_to :html, :js, :json
 
-  before_action :initialize_service_request,  except: [:approve_epic_rights, :push_to_epic, :push_to_epic_status]
-  before_action :authorize_identity,          except: [:approve_epic_rights, :push_to_epic, :push_to_epic_status]
-  before_action :find_protocol,               only:   [:edit, :update, :show]
+  before_action :initialize_service_request,  only: [:show, :new, :create, :edit, :update, :update_protocol_type]
```

Comment:
The `validate_rmid` and `get_study_type_note` routes were running initialize_service_request and authorize_identity when they shouldn't have been. I changed the except to and only to prevent that from happening by mistake in the future with new routes